rest-service/manager_rest/rest/resources_v3/events.py

```diff
@@ -50,22 +51,37 @@ def post(self):
             check_user_action_allowed('set_timestamp')
 
         sm = get_storage_manager()
-        exc = current_execution._get_current_object()
-        if exc is None:
-            exc_id = request_dict.get('execution_id')
-            if exc_id is None:
+
+        exc_id = request_dict.get('execution_id')
+        eg_id = request_dict.get('execution_group_id')
+
+        if exc_id and eg_id:
+            raise manager_exceptions.ConflictError(
+                'Both execution_id and execution_group_id were provided.')
```

Comment:
```suggestion
                'Cannot provide both execution_id and execution_group_id.')
```
In general, error messages should talk about an error, not just make a statement of fact